### PR TITLE
Updated QoS & deleted intra_process_comms (No LC node))

### PR DESCRIPTION
**Pull request by igorrecioh**
### Dashing migration (avoiding warnings):
- Updated QoS policies -> Publishers & Subscribers (changes parameter order)
- Deleted intra_process_comms variable as it is not a LifeCycle Node

### CODE REVIEW

File: hros_cognition_mara_components/src/HROSCognitionMaraComponents.cpp [hunk 1]
```diff
@@ -58,16 +56,16 @@ HROSCognitionMaraComponentsNode::HROSCognitionMaraComponentsNode(const std::stri
...
                                 stateCallback(motor_name, msg->velocity, msg->position, msg->effort);
-                              },rmw_qos_profile_sensor_data);
+                              });
     motor_state_subcriptions_.push_back(subscriber);
     std::cout << "Subscribe at " << motor_name << std::endl;
 
     motor_name = id + "goal" + axis;
 
-    auto publisher_command = this->create_publisher<hrim_actuator_rotaryservo_msgs::msg::GoalRotaryServo>(motor_name, rmw_qos_profile_sensor_data);
+    auto publisher_command = this->create_publisher<hrim_actuator_rotaryservo_msgs::msg::GoalRotaryServo>(motor_name, rclcpp::SensorDataQoS(rclcpp::KeepLast(1)));
```
Comment (nzlz) [inline hros_cognition_mara_components/src/HROSCognitionMaraComponents.cpp]: You are adding `depth=1`, just warning in case its not intented.

**Comment by LanderU** [comment]
CI passed, can we merge this or are you going to include more commits here?

File: hros_cognition_mara_components/src/HROSCognitionMaraComponents.cpp (hunk 1 above)
Comment (igorrecioh) [reply hros_cognition_mara_components/src/HROSCognitionMaraComponents.cpp]: Thanks @nzlz ! Yes, it is intended just to follow a common standard. Do you think that we should change depth value in this case?

**Comment by igorrecioh** [comment]
Let's solve the depth doubt with @nzlz and then we can merge. I will do PR for each package if it's ok for you, just to split a little bit

**Comment by LanderU** [comment]
> I will do PR for each package if it's ok for you, just to split a little bit

+1

File: hros_cognition_mara_components/src/HROSCognitionMaraComponents.cpp (hunk 1 above)
Comment (nzlz) [reply hros_cognition_mara_components/src/HROSCognitionMaraComponents.cpp]: Nothing against the change, I think it won't hurt.

**Comment by YueErro** [comment]
Regarding QoS taken from [Release-Dashing-Diademata](https://index.ros.org//doc/ros2/Releases/Release-Dashing-Diademata/#rclcpp):

>If you have no idea what depth to use and don’t care right now (maybe just prototyping), then we recommend using 10, as that was the default before and should preserve existing behavior.
More in depth documentation about how to select an appropriate depth is forthcoming.
This is an example of a slightly more involved change to avoid the newly deprecated API’s:
```diff
- // Creates a latched topic
- rmw_qos_profile_t qos = rmw_qos_profile_default;
- qos.depth = 1;
- qos.durability = RMW_QOS_POLICY_DURABILITY_TRANSIENT_LOCAL;
-
  model_xml_.data = model_xml;
  node_handle->declare_parameter("robot_description", model_xml);
  description_pub_ = node_handle->create_publisher<std_msgs::msg::String>(
-   "robot_description", qos);
+   "robot_description",
+   // Transient local is similar to latching in ROS 1.
+   rclcpp::QoS(1).transient_local());
```
I understand that if we have previoulsy defined `qos_state.depth = 1;` we should use `rclcpp::SensorDataQoS(rclcpp::KeepLast(1))` for `rmw_qos_profile_sensor_data` and `rclcpp::QoS(1)` for `rmw_qos_profile_sensor`.
If it is not defined it should be `(10)` instead of `(1)`.

File: hros_cognition_mara_components/src/HROSCognitionMaraComponents.cpp (hunk 1 above)
Comment (igorrecioh) [reply hros_cognition_mara_components/src/HROSCognitionMaraComponents.cpp]: Nice, thank you @nzlz!

**Comment by igorrecioh** [comment]
LGTM!

**Pull request merged**